### PR TITLE
add autocomplete off to email field in site settings

### PR DESCRIPTION
**Pull request by mmarkusik**
[SALEOR-1133](https://app.clickup.com/t/2549495/SALEOR-1133)

I want to merge this change because the autocomplete in the email field is unnecessary.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1133 Disable autocomplete on mail configuration](https://app.clickup.com/t/2549495/SALEOR-1133)

File: src/siteSettings/components/SiteSettingsMailing/SiteSettingsMailing.tsx [hunk 1]
```diff
@@ -114,6 +114,7 @@ const SiteSettingsMailing: React.FC<SiteSettingsMailingProps> = props => {
           disabled={disabled}
           error={!!formErrors.customerSetPasswordUrl}
           fullWidth
+          autoComplete="off"
```
Comment (patrys) [inline src/siteSettings/components/SiteSettingsMailing/SiteSettingsMailing.tsx:117]: Could we also add `type="url"`?

Comment (mmarkusik) [reply src/siteSettings/components/SiteSettingsMailing/SiteSettingsMailing.tsx:117]: yis

**Review by krzysztofwolski — APPROVED** [review]
LGTM

**Review by orzechdev — APPROVED** [review]

**Pull request merged**